nexus/src/app/instance.rs

```diff
@@ -799,220 +797,6 @@ impl super::Nexus {
         Ok(disk)
     }
 
-    /// Create a network interface attached to the provided instance.
```

Comment:
I relocated the network interface logic in its own file. We break this resource at the data store layer, might as well be consistent with it.